lib/OrdinaryDiffEqStabilizedIRK/src/algorithms.jl

```diff
@@ -1,3 +1,12 @@
+@doc generic_solver_docstring("Description TBD",
+    "IRKC",
+    "Stabalized Implicit Runge Kutta method.",
+    "REF TBD",
```

Comment:
@ChrisRackauckas ref and description for this implicit stabilized method.

Reply:
@ChrisRackauckas you missed this one.

Reply:
Stabilized Explicit Runge-Kutta methods, like Runge-Kutta-Chebyshev methods and ROCK methods, are explicit methods which chain together many stages in a specific way to get large stability regions. they are made in such a way to converge to a large stability region, and thus suitable to stiff equations. However, they converge to having a large stability region in the direction of the negative real axis, and thus are only stable on a subset of stiff equations which are not dominated by large complex eigenvalues in the Jacobian. Stabilized implicit methods try to mitigate this problem by being an IMEX type scheme, requiring a SplitODEProblem where the splitting is designed to treat the large complex eigenvalues implicitly while treating the large real eigenvalues using a fast explicit stabilized RK type of method.

Reply:
Still needs a ref.